src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -1623,6 +1623,11 @@ def test_containerapp_create_with_vnet_yaml(self, resource_group):
 
 
 class ContainerappOtherPropertyTests(ScenarioTest):
+    def test_containerapp_get_customdomainverificationid(self):
+        self.cmd('configure --defaults location={}'.format(TEST_LOCATION))
+
+        self.cmd(f'containerapp show-custom-domain-verification-id', expect_failure=False)
```

Comment:
could you also add command to use it bind domain?

Reply:
Done.